Features

ros-a opened the pull request:
Hi Rayna!
If you could have a look whenever you have time that would be great.
All the images are placeholders so don't pay attention to that.
I'm also still going to add titles etc. to the projects. 
Cheers!

ros-a committed:
Made basic html doc

ros-a committed:
made basic structure of html doc

ros-a committed:
set up basic css file

ros-a committed:
added placeholder photos of projects

ros-a committed:
plant placeholder photo

ros-a committed:
added styles

ros-a committed:
adding media files

ros-a committed:
added responsiveness

ros-a committed:
added responsiveness

ros-a committed:
Merge pull request #1 from ros-a/in-progress

In progress

ros-a committed:
added more responsiveness

ros-a committed:
added CSS

ros-a committed:
added some links and tidied a bit

ros-a committed:
changed the bits you commented on Sam

ros-a committed:
Merge pull request #2 from ros-a/in-progress

ros-a committed:
added effects to the portfolio projects

norayna commented:
portfolio items only have a picture - it would be ideal to have a name and description and maybe even github link for each one, because you don't have a separate page for individual projects. i'd say to add:
- name of each project - for sure
- description - if possible
- github link - if possible

norayna commented:
on mobile - portfolio images are slightly off to one side:
![image](https://user-images.githubusercontent.com/25844355/125773138-00ce17a8-cf97-4b82-83d3-a742d3c686a3.png)
i assume they are supposed to be centered?

norayna commented on index.html:
```diff
@@ -0,0 +1,82 @@
+        <section id="projects">
+            <h2 class="projects-title">projects</h2>
+            <div class="projects-container">
+                <a href="#" class="projects project1">
+                    <div class="project-images project-image1"></div>
+                </a>
+                <a href="#" class="projects project2">
+                    <div class="project-images project-image2"></div>
+                </a>
+                <a href="#" class="projects project3">
+                    <div class="project-images project-image3"></div>
+                </a>
+                <a href="#" class="projects project4">
+                    <div class="project-images project-image4"></div>
+                </a>
+                <a href="#" class="projects project5">
+                    <div class="project-images project-image4"></div>
+                </a>
+                <a href="#" class="projects project6">
```
as far as i can tell the classes project1 to project6 and project-image1 to project-image6 are not doing anything? if so, you can remove them. also if there are any other classes that are not used for anything they should be removed as well - we create css classes only when they are needed

if they are doing something - bare in mind the class on the 5th project image is project-image4 - probably a mistake

ros-a committed:
added effects to portfolio projects

norayna reviewed (changes requested):
just the few comments there already

ros-a committed:
added hover effects to projects

ros-a committed:
tidied a little and changed some responsiveness

ros-a committed:
Merge pull request #4 from ros-a/in-progress

In progress

ros-a commented:
Hi! I changed the bits you commented on. I also added titles to the projects, but as soon as I have any projects, I'd like them to open to a new page, where I'll then have a description and Github link :) so for now it's just the photo with a title but that will change as soon as I've finished my first project to put on there!

norayna reviewed (approved):
approved 🧅
